Shared/Networking/Model.swift

```diff
@@ -62,25 +90,27 @@ struct TiltifyCampaignRewardImage: Codable {
 
 struct TiltifyCampaignReward: Codable {
     let id: Int
+    let publicId: UUID
     let name: String
     let description: String
     let amount: TiltifyAmount
     let image: TiltifyCampaignRewardImage?
 }
 
 struct TiltifyCampaign: Codable {
-    let publicId: String
+    let publicId: UUID
     let avatar: TiltifyAvatar?
     let goal: TiltifyAmount
     let milestones: [TiltifyMilestone]
     let slug: String
     let status: String
     let team: String?
+    let user: TiltifyUser
     let description: String
     let totalAmountRaised: TiltifyAmount
     let name: String
     let originalGoal: TiltifyAmount
-    let rewards: [TiltifyCampaignReward]
+    let rewards: [TiltifyCampaignReward]    
```

Comment:
Is the whitespace at the end intentional?

Reply:
Nope!